lib/utils/zero-config.js

```diff
@@ -1,8 +1,8 @@
 function getEnvFromOptionsAndMode(mode, optionsObject) {
     const { dev, prod } = optionsObject;
```

Comment:
```js
function getEnvFromOptionsAndMode(mode, {dev, prod}) {
```

Reply:
That's not my favourite suggestion. I would keep it like this. Destructuring is fine when there's only one single object passed to the function. Here, we don't know what's inside the second parameter (variable names help)

Reply:
Yeah, makes sense.